doc/internal/man3/evp_pkey_export_to_provider.pod

```diff
@@ -0,0 +1,72 @@
+=pod
+
+=head1 NAME
+
+evp_pkey_export_to_provider, evp_pkey_upgrade_to_provider
+- internal EVP_PKEY support functions for providers
+
+=head1 SYNOPSIS
+
+ /* Only for EVP source */
+ #include "evp_local.h"
+
+ void *evp_pkey_export_to_provider(EVP_PKEY *pk, OPENSSL_CTX *libctx,
+                                   EVP_KEYMGMT **keymgmt,
+                                   const char *propquery);
+ void *evp_pkey_upgrade_to_provider(EVP_PKEY *pk, OPENSSL_CTX *libctx,
+                                    EVP_KEYMGMT **keymgmt,
+                                    const char *propquery);
+
+=head1 DESCRIPTION
+
+evp_pkey_export_to_provider() exports the "origin" key contained in I<pk>
```

Comment:
Its not clear what "origin" key is here (to someone new reading this)

Reply:
Have a look at `doc/internal/man7/EVP_PKEY.pod`.  I'm adding a line here to refer to it.